src/ILCompiler.Compiler/src/Compiler/Compilation.cs

```diff
@@ -204,9 +205,24 @@ private void AddCompilationRoots()
                 AddCompilationRootsForRuntimeExports(module);
            }
 
+            AddCompilationRootsForBootstrap();
```

Comment:
I generally don't care about naming on things like this.. I defer to Michal who tends to have better naming instincts.